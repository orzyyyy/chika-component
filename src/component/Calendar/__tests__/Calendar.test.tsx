import React from 'react';
import { mount } from 'enzyme';
import 'nino-cli/scripts/setup';

let Calendar;
switch (process.env.LIB_DIR) {
  case 'lib':
    Calendar = require('../../../../lib/component/Calendar').default;
    break;
  case 'dist':
    Calendar = require('../../../../dist/lib/Calendar').default;
    break;
  default:
    Calendar = require('..').default;
    break;
}

describe('Calendar', () => {
  it('render correctly', () => {
    const wrapper = mount(<Calendar />);
    expect(wrapper).toMatchSnapshot();
  });
});
