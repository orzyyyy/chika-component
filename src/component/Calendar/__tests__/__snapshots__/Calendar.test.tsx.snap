// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar render correctly 1`] = `
<Calendar>
  <div
    className="Calendar"
  >
    <div
      className="container"
      key="list_div_0"
      style={
        Object {
          "opacity": 0,
          "transform": "translate3d(400%, 0 , 0)",
        }
      }
    >
      <table
        className="week-name"
      >
        <tbody
          key="body_tbody_-1"
        >
          <tr>
            <td
              key="week_td_0"
            >
              <span>
                日
              </span>
            </td>
            <td
              key="week_td_1"
            >
              <span>
                一
              </span>
            </td>
            <td
              key="week_td_2"
            >
              <span>
                二
              </span>
            </td>
            <td
              key="week_td_3"
            >
              <span>
                三
              </span>
            </td>
            <td
              key="week_td_4"
            >
              <span>
                四
              </span>
            </td>
            <td
              key="week_td_5"
            >
              <span>
                五
              </span>
            </td>
            <td
              key="week_td_6"
            >
              <span>
                六
              </span>
            </td>
          </tr>
        </tbody>
        <tbody
          key="body_tbody_0"
        >
          <tr />
        </tbody>
        <tbody
          key="body_tbody_1"
        >
          <tr />
        </tbody>
      </table>
    </div>
    <div
      className="container"
      key="list_div_1"
      style={
        Object {
          "opacity": 0,
          "transform": "translate3d(300%, 0 , 0)",
        }
      }
    >
      <table
        className="week-name"
      >
        <tbody
          key="body_tbody_-1"
        >
          <tr>
            <td
              key="week_td_0"
            >
              <span>
                日
              </span>
            </td>
            <td
              key="week_td_1"
            >
              <span>
                一
              </span>
            </td>
            <td
              key="week_td_2"
            >
              <span>
                二
              </span>
            </td>
            <td
              key="week_td_3"
            >
              <span>
                三
              </span>
            </td>
            <td
              key="week_td_4"
            >
              <span>
                四
              </span>
            </td>
            <td
              key="week_td_5"
            >
              <span>
                五
              </span>
            </td>
            <td
              key="week_td_6"
            >
              <span>
                六
              </span>
            </td>
          </tr>
        </tbody>
        <tbody
          key="body_tbody_0"
        >
          <tr />
        </tbody>
        <tbody
          key="body_tbody_1"
        >
          <tr />
        </tbody>
      </table>
    </div>
    <div
      className="container"
      key="list_div_2"
      style={
        Object {
          "opacity": 0,
          "transform": "translate3d(200%, 0 , 0)",
        }
      }
    >
      <table
        className="week-name"
      >
        <tbody
          key="body_tbody_-1"
        >
          <tr>
            <td
              key="week_td_0"
            >
              <span>
                日
              </span>
            </td>
            <td
              key="week_td_1"
            >
              <span>
                一
              </span>
            </td>
            <td
              key="week_td_2"
            >
              <span>
                二
              </span>
            </td>
            <td
              key="week_td_3"
            >
              <span>
                三
              </span>
            </td>
            <td
              key="week_td_4"
            >
              <span>
                四
              </span>
            </td>
            <td
              key="week_td_5"
            >
              <span>
                五
              </span>
            </td>
            <td
              key="week_td_6"
            >
              <span>
                六
              </span>
            </td>
          </tr>
        </tbody>
        <tbody
          key="body_tbody_0"
        >
          <tr />
        </tbody>
        <tbody
          key="body_tbody_1"
        >
          <tr />
        </tbody>
      </table>
    </div>
    <div
      className="container"
      key="list_div_3"
      style={
        Object {
          "opacity": 0,
          "transform": "translate3d(100%, 0 , 0)",
        }
      }
    >
      <table
        className="week-name"
      >
        <tbody
          key="body_tbody_-1"
        >
          <tr>
            <td
              key="week_td_0"
            >
              <span>
                日
              </span>
            </td>
            <td
              key="week_td_1"
            >
              <span>
                一
              </span>
            </td>
            <td
              key="week_td_2"
            >
              <span>
                二
              </span>
            </td>
            <td
              key="week_td_3"
            >
              <span>
                三
              </span>
            </td>
            <td
              key="week_td_4"
            >
              <span>
                四
              </span>
            </td>
            <td
              key="week_td_5"
            >
              <span>
                五
              </span>
            </td>
            <td
              key="week_td_6"
            >
              <span>
                六
              </span>
            </td>
          </tr>
        </tbody>
        <tbody
          key="body_tbody_0"
        >
          <tr />
        </tbody>
        <tbody
          key="body_tbody_1"
        >
          <tr />
        </tbody>
      </table>
    </div>
    <div
      className="container"
      key="list_div_4"
      style={
        Object {
          "opacity": 1,
          "transform": "translate3d(0%, 0 , 0)",
        }
      }
    >
      <table
        className="week-name"
      >
        <tbody
          key="body_tbody_-1"
        >
          <tr>
            <td
              key="week_td_0"
            >
              <span>
                日
              </span>
            </td>
            <td
              key="week_td_1"
            >
              <span>
                一
              </span>
            </td>
            <td
              key="week_td_2"
            >
              <span>
                二
              </span>
            </td>
            <td
              key="week_td_3"
            >
              <span>
                三
              </span>
            </td>
            <td
              key="week_td_4"
            >
              <span>
                四
              </span>
            </td>
            <td
              key="week_td_5"
            >
              <span>
                五
              </span>
            </td>
            <td
              key="week_td_6"
            >
              <span>
                六
              </span>
            </td>
          </tr>
        </tbody>
        <tbody
          key="body_tbody_0"
        >
          <tr />
        </tbody>
        <tbody
          key="body_tbody_1"
        >
          <tr />
        </tbody>
      </table>
    </div>
    <div
      className="container"
      key="list_div_5"
      style={
        Object {
          "opacity": 0,
          "transform": "translate3d(-100%, 0 , 0)",
        }
      }
    >
      <table
        className="week-name"
      >
        <tbody
          key="body_tbody_-1"
        >
          <tr>
            <td
              key="week_td_0"
            >
              <span>
                日
              </span>
            </td>
            <td
              key="week_td_1"
            >
              <span>
                一
              </span>
            </td>
            <td
              key="week_td_2"
            >
              <span>
                二
              </span>
            </td>
            <td
              key="week_td_3"
            >
              <span>
                三
              </span>
            </td>
            <td
              key="week_td_4"
            >
              <span>
                四
              </span>
            </td>
            <td
              key="week_td_5"
            >
              <span>
                五
              </span>
            </td>
            <td
              key="week_td_6"
            >
              <span>
                六
              </span>
            </td>
          </tr>
        </tbody>
        <tbody
          key="body_tbody_0"
        >
          <tr />
        </tbody>
        <tbody
          key="body_tbody_1"
        >
          <tr />
        </tbody>
      </table>
    </div>
  </div>
</Calendar>
`;
